### DISCUSSION TITLE
[Parseq example] Movement: sliding forwards (translation z axis, linear, continuous, amount relative to previous frame)

### DISCUSSION THREAD

**Discussion by rewbs** [rewbs/sd-parseq#117] [Parseq example library]
https://github.com/rewbs/sd-parseq/assets/74455/1a83d395-e22c-4a4b-bb48-5d91afb83730

Translation along the z axis by a constant increment relative to the previous frame using prev_computed_value. Gives the impression that the viewer is "sliding forwards".
        
🚀 [Parseq direct link](https://sd-parseq.web.app/?importRemote=GUWZvNjsIQOWlB5HVb2uT3vEg4u1%2Fdoc-9dfa16c4-3877-4f30-986b-41603ba72515-1685685499504.json&token=0d62bf0d-b321-4aef-ac31-dca1b051a360)
⬇️ [Parsec doc](https://gist.github.com/rewbs/46e9d005656c2ecc69df16868a462f20)
📃 [Deforum settings](https://gist.github.com/rewbs/35a80c3ebb499212bc57dd5bc2d7482b)
ℹ️ Post processing notes: x3 FILM (20fps to 60fps)

**Comment by shlomitgueta** [comment]
I would like to see more parsec doc with other functions. I love your tutorials.Thank you
